src/main/java/dao/impl/OrderDaoImpl.java

```diff
@@ -1,15 +1,101 @@
 package dao.impl;
 
 import dao.OrderDao;
-import db.Storage;
 import model.Order;
+import model.Product;
+import model.User;
+import org.apache.log4j.Logger;
+import util.DBConnection;
 
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
 import java.util.List;
 import java.util.Optional;
 
 public class OrderDaoImpl implements OrderDao {
+
+    private static final Logger logger = Logger.getLogger(UserDaoImpl.class);
+
+    @Override
+    public void addOrder(Order value, User user) {
```

Comment:
try to simplify this method

Reply:
> try to simplify this method

I need your help with this :)
In this logic: we put a new order in the 'user_order' table. The id for a new order will be generated automatically, we need to know this id and therefore we select the order id from the 'user_order' table. Then we put the list of products in the 'order_product' table.